### PR TITLE
add warning to hll/tdigest that the format is subject to change in the future

### PR DESCRIPTION
**Pull request by ziham1531991**

### CODE REVIEW

**Comment by Jak-MS** [comment]
@shsagir 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/kusto/query/hll-aggfunction.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ Read about the [underlying algorithm (*H*yper*L*og*L*og) and the estimation accu
 >- Use the [hll_merge](hllmergefunction.md) function to merge the results of multiple `hll()` functions.
 >- Use the [dcount_hll](dcount-hllfunction.md) function to calculate the number of distinct values from the output of the `hll()` or `hll_merge` functions.
 
+> [!WARNING]
+>The results of hll(), hll_if() and hll_merge() can be stored and later retrieved (for example, to create a daily unique users summary that can be later be used to calculate weekly counts). 
+> However, the precise binary representation of these results is undocumented and may change over time. There’s no guarantee that these functions return the same output for the same input, and one should not take a dependency on them.
```
Comment (shsagir) [inline data-explorer/kusto/query/hll-aggfunction.md:23]:
```suggestion
> [!IMPORTANT]
>The results of hll(), hll_if(), and hll_merge() can be stored and later retrieved. For example, you may want to create a daily unique users summary, which can then be used to calculate weekly counts.
> However, the precise binary representation of these results may change over time. There's no guarantee that these functions will produce identical results for identical inputs, and therefore we don't advise relying on them.
```

File: data-explorer/kusto/query/hll-if-aggregation-function.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ Read about the [underlying algorithm (*H*yper*L*og*L*og) and the estimation accu
 
 [!INCLUDE [data-explorer-agg-function-summarize-note](../../includes/data-explorer-agg-function-summarize-note.md)]
 
+> [!WARNING]
+>The results of hll(), hll_if() and hll_merge() can be stored and later retrieved (for example, to create a daily unique users summary that can be later be used to calculate weekly counts). 
+> However, the precise binary representation of these results is undocumented and may change over time. There’s no guarantee that these functions return the same output for the same input, and one should not take a dependency on them.
```
Comment (shsagir) [inline data-explorer/kusto/query/hll-if-aggregation-function.md:18]:
```suggestion
> [!IMPORTANT]
> The results of hll(), hll_if(), and hll_merge() can be stored and later retrieved. For example, you may want to create a daily unique users summary, which can then be used to calculate weekly counts.
> However, the precise binary representation of these results may change over time. There's no guarantee that these functions will produce identical results for identical inputs, and therefore we don't advise relying on them.
```

File: data-explorer/kusto/query/hll-merge-aggfunction.md [hunk 3]
```diff
@@ -13,6 +13,10 @@ Merges HLL results across the group into a single HLL value.
 
 For more information, see the [underlying algorithm (*H*yper*L*og*L*og) and estimation accuracy](#estimation-accuracy).
 
+> [!WARNING]
+>The results of hll() and hll_merge() can be stored and later retrieved (for example, to create a daily unique users summary that can be later be used to calculate weekly counts). 
+> However, the precise binary representation of these results is undocumented and may change over time. There’s no guarantee that these functions return the same output for the same input, and one should not take a dependency on them.
```
Comment (shsagir) [inline data-explorer/kusto/query/hll-merge-aggfunction.md:18]:
```suggestion
> [!IMPORTANT]
> The results of hll(), hll_if(), and hll_merge() can be stored and later retrieved. For example, you may want to create a daily unique users summary, which can then be used to calculate weekly counts.
> However, the precise binary representation of these results may change over time. There's no guarantee that these functions will produce identical results for identical inputs, and therefore we don't advise relying on them.
```

File: data-explorer/kusto/query/hllmergefunction.md [hunk 4]
```diff
@@ -11,6 +11,10 @@ Merges HLL results. This is the scalar version of the aggregate version [`hll_me
 
 Read about the [underlying algorithm (*H*yper*L*og*L*og) and estimation accuracy](#estimation-accuracy).
 
+> [!WARNING]
+>The results of hll(), hll_if() and hll_merge() can be stored and later retrieved (for example, to create a daily unique users summary that can be later be used to calculate weekly counts). 
+> However, the precise binary representation of these results is undocumented and may change over time. There’s no guarantee that these functions return the same output for the same input, and one should not take a dependency on them.
```
Comment (shsagir) [inline data-explorer/kusto/query/hllmergefunction.md:16]:
```suggestion
> [!IMPORTANT]
> The results of hll(), hll_if(), and hll_merge() can be stored and later retrieved. For example, you may want to create a daily unique users summary, which can then be used to calculate weekly counts.
> However, the precise binary representation of these results may change over time. There's no guarantee that these functions will produce identical results for identical inputs, and therefore we don't advise relying on them.
```

File: data-explorer/kusto/query/merge-tdigest-function.md [hunk 5]
```diff
@@ -13,6 +13,10 @@ Read more about the underlying algorithm (T-Digest) and the estimated error [her
 
 > The `merge_tdigest()` and `tdigest_merge()` functions are equivalent
 
+> [!WARNING]
+>The results of tdigest() and tdigest_merge() can be stored and later retrieved (for example, to create a daily percentiles summary that can be later be used to calculate weekly percentiles). 
+> However, the precise binary representation of these results is undocumented and may change over time. There’s no guarantee that these functions return the same output for the same input, and one should not take a dependency on them.
```
Comment (shsagir) [inline data-explorer/kusto/query/merge-tdigest-function.md:18]:
```suggestion
> [!IMPORTANT]
>The results of tdigest() and tdigest_merge() can be stored and later retrieved. For example, you may want to create daily percentiles summary, which can then be used to calculate weekly percentiles.
> However, the precise binary representation of these results may change over time. There's no guarantee that these functions will produce identical results for identical inputs, and therefore we don't advise relying on them.
```

File: data-explorer/kusto/query/tdigest-aggfunction.md [hunk 6]
```diff
@@ -13,6 +13,10 @@ Calculates the intermediate results of [`percentiles()`](percentiles-aggfunction
 
 For more information, see the [underlying algorithm (T-Digest) and the estimated error](percentiles-aggfunction.md#estimation-error-in-percentiles).
 
+> [!WARNING]
+>The results of tdigest() and tdigest_merge() can be stored and later retrieved (for example, to create a daily percentiles summary that can be later be used to calculate weekly percentiles). 
+> However, the precise binary representation of these results is undocumented and may change over time. There’s no guarantee that these functions return the same output for the same input, and one should not take a dependency on them.
```
Comment (shsagir) [inline data-explorer/kusto/query/tdigest-aggfunction.md:18]:
```suggestion
> [!IMPORTANT]
>The results of tdigest() and tdigest_merge() can be stored and later retrieved. For example, you may want to create daily percentiles summary, which can then be used to calculate weekly percentiles.
> However, the precise binary representation of these results may change over time. There's no guarantee that these functions will produce identical results for identical inputs, and therefore we don't advise relying on them.
```

File: data-explorer/kusto/query/tdigest-merge-aggfunction.md [hunk 7]
```diff
@@ -15,6 +15,10 @@ For more information about the underlying algorithm (T-Digest) and the estimated
 
 > The `tdigest_merge()` and `merge_tdigest()` functions are equivalent
 
+> [!WARNING]
+>The results of tdigest() and tdigest_merge() can be stored and later retrieved (for example, to create a daily percentiles summary that can be later be used to calculate weekly percentiles). 
+> However, the precise binary representation of these results is undocumented and may change over time. There’s no guarantee that these functions return the same output for the same input, and one should not take a dependency on them.
```
Comment (shsagir) [inline data-explorer/kusto/query/tdigest-merge-aggfunction.md:20]:
```suggestion
> [!IMPORTANT]
>The results of tdigest() and tdigest_merge() can be stored and later retrieved. For example, you may want to create daily percentiles summary, which can then be used to calculate weekly percentiles.
> However, the precise binary representation of these results may change over time. There's no guarantee that these functions will produce identical results for identical inputs, and therefore we don't advise relying on them.
```

File: data-explorer/kusto/query/usinghlltdigest.md [hunk 8]
```diff
@@ -15,6 +15,9 @@ Partitioning queries as described is easy for simple aggregates, such as `count(
 
 The following examples show how to use `hll`/`tdigest` and demonstrate that using these commands is highly performant in some scenarios:
 
+> [!WARNING]
+> The precise binary representation of hll(), hll_merge(), tdigest() and tdigest_merge() results is undocumented and may change over time. There’s no guarantee that these functions return the same output for the same input, and one should not take a dependency on them.
```
Comment (shsagir) [inline data-explorer/kusto/query/usinghlltdigest.md:19]:
```suggestion
> [!IMPORTANT]
> The precise binary representation of hll(), hll_if(), hll_merge(), tdigest(), and tdigest_merge() results may change over time. There's no guarantee that these functions will produce identical results for identical inputs, and therefore we don't advise relying on them.
```

**Review by shsagir — CHANGES_REQUESTED** [review]

**Review by shsagir — APPROVED** [review]

**Comment by ziham1531991** [comment]
#sign-off

**Comment by shsagir** [comment]
#sign-off

**Pull request merged**